doc/source/reference/commands/status.rst

```diff
@@ -147,3 +149,39 @@ values:
    * - ``version``
      - The version of the Groonga process.
      - ``5.0.7``
+
+   * - ``features``
+     - .. versionadded:: 10.0.1
+     
+       The list of features that Groonga has and whether they are enabled or not.
+     - .. code-block::
+
+          {
+             "nfkc": true,
+             "mecab": true,
+             "message_pack": true,
+             "mruby": true,
+             "onigmo": true,
+             "zlib": true,
+             "lz4": false,
+             "zstandard": false,
+             "kqueue": false,
+             "epoll": true,
+             "poll": false,
+             "rapidjson": false,
+             "apache_arrow": false,
+             "xxhash": false
+          }
+  
+   * - ``apache_arrow``
+     - .. versionadded:: 10.0.1
+     
+       The version of Apache Arrow that Groonga uses. This item displays only when Apache Arrow is enabled in Groonga. 
```

Comment:
We may add not only version information but also more information later.

Reply:
Ok, I have fixed it.